openmetadata-ui/src/main/resources/ui/src/rest/searchAPI.ts

```diff
@@ -184,14 +184,15 @@ export const rawSearchQuery = <
       : queryWithSlash
     : '';
 
+  const apiUrl = `/search/query?q=${apiQuery}${filters ?? ''}`;
```

Comment:
Let's utilize the param option and pass `q` in params only.

Reply:
We are passing as param from this function only, for all other places we are passing in url. 
So i needed a consistent approach, so decided to change at one place to reduce impact.

Reply:
We are not recommending passing params in the URL directly, so would prefer the params approach here.
In other places, we can do refactoring later.